customconfig/utils.go

```diff
@@ -3,9 +3,55 @@ package customconfig
 import (
 	"crypto/sha256"
 	"encoding/hex"
+	"fmt"
+	"log"
+	"math/rand"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
 )
 
 func hash(s string) string {
 	sha := sha256.Sum256([]byte(s))
 	return hex.EncodeToString(sha[:])
 }
+
+func retry(retryFunc func() error, minutes int) error {
+	return retryTime(retryFunc, minutes)
+}
+
+func retryTime(retryFunc func() error, minutes int) error {
+	wait := 1
+	return resource.Retry(time.Duration(minutes)*time.Minute, func() *resource.RetryError {
```

Comment:
fwiw, `Deprecated: Please use RetryContext to ensure proper plugin shutdown`
https://godoc.org/github.com/hashicorp/terraform-plugin-sdk/helper/resource#Retry

Reply:
Using latest 1.13.0: `customconfig/utils.go:28:9: undefined: resource.RetryContext`

Is this coming from v2? (which only has a RC https://github.com/hashicorp/terraform-plugin-sdk/releases)